website/docs/reference/build-info.md

```diff
@@ -51,6 +51,8 @@ object BuildInfo {
   val mainClass = Some("Main")
   /** Project version */
   val projectVersion = None
+  /** Scala-cli version used for the compilation */
+  val scalaCliVersion = Some("1.1.1-SNAPSHOT")
```

Comment:
mental note, that this doc should get updated with some more modern values =)
```suggestion
  val scalaCliVersion = Some("1.7.1")
```

Reply:
@Gedochao I fixed the tests in SourceGeneratorTests, I hope that is the only place. 
Also, since the constants file was not accessible there, i hardcoded the version 1.1.1-SNAPSHOT in the test. Is it ok?

Reply:
@Gedochao Could you review this once again. I think there are some test failures, will have a look at it once again. But is the other changes ok?